config/metadata/rulesMeta.yaml

```
@@ -651,6 +651,12 @@ groups:
         summary: is the comparison operator to use.
         description: >
           The comparison operator to use. String comparisons are case-sensitive.
+          For most cases, use the negative operators (`!=`, `does-not-contain`, and
+          `not-exists`) in a rule with a scope of "span". WARNING: A negative operator
+          in a rule with a scope of "trace" (the default) can cause the rule to match
+          a trace if *any* span in the trace matches the negative condition, for example
```

We only use bold in docs, not italics.
```suggestion
          a trace if **any** span in the trace matches the negative condition. For example,
```